nncf/experimental/torch/fx/model_transformer.py

```diff
@@ -97,7 +98,23 @@ def _apply_model_extraction(
         # TODO(dlyakhov): reduce memory consumption by
         # more optimal splitting implementation.
         splitted_gm = split_by_tags(model, tags)
-        return splitted_gm.extracted
+
+        extracted_model = splitted_gm.extracted
```

Comment:
Please, add test case that will cover new case and a case with conv too, like 
```
def test_extracted_model(syntetic_model, extracted_ref):
...
```

Reply:
This test is expected to be in a separate PR. swin_v2_s have this problematic const linear inside, is presented in the conformance test. Should it be enough for merge to experimental dir??

Reply:
Unit test is presented, please check